### PR TITLE
Updating the min agent version - There is a agent side change done fo…

### PR DESCRIPTION
**Pull request by chshrikh**
…r showing the warning message in case machine group is used in task

### CODE REVIEW

File: Tasks/DeployVisualStudioTestAgent/task.json [hunk 1]
```diff
@@ -13,12 +13,12 @@
...
         "Major": 1,
         "Minor": 0,
-        "Patch": 28
+        "Patch": 29
     },
     "demands": [
 
     ],
-    "minimumAgentVersion": "1.97.0",
+    "minimumAgentVersion": "1.101.0",
```
Comment (acesiddhu) [inline Tasks/DeployVisualStudioTestAgent/task.json]: we shuld check if this will work even on hosted agents.

Comment (chshrikh) [inline Tasks/DeployVisualStudioTestAgent/task.json]: I checked with RM team, it should work fine

**Comment by KrishnaAdityaB** [comment]
:shipit:

**Comment by acesiddhu** [comment]
ok then it looks fine

**Pull request merged**